metrics/serve_test.go

```diff
@@ -176,8 +305,8 @@ func TestRunMetricsGradualChange(t *testing.T) {
 
 	time.Sleep(2 * time.Second)
 	currentCount := countSeries(t, reg)
-	expectedInitialCount := currentCount
-	assert.Equal(t, expectedInitialCount, currentCount, "Initial series count should be minSeriesCount %d but got %d", expectedInitialCount, currentCount)
+	fmt.Println("seriesCount: ", currentCount)
```

Comment:
Might want to remove this fmt?

Reply:
Let's keep it, it's everywhere else in the tests, let's fix globally if needed (: OK?